cve_bin_tool/cli.py

```diff
@@ -29,6 +29,7 @@
 from .extractor import Extractor
 from .strings import Strings
 from .file import is_binary
+from .output_engine import OutputEngine
```

Comment:
I know the tests are like this, with the filename and the class name similar-but-different, but we already have VersionSignature.py, so it's not a big deal to have OutputEngine.py.